packages/ag-charts-community/src/chart/series/cartesian/scaling.ts

```diff
@@ -45,3 +45,13 @@ export function areScalingEqual(a: Scaling | undefined, b: Scaling | undefined):
     }
     return false;
 }
+
+export function isScaleValid(scale?: Scaling) {
+    if (scale == null) return false;
+    if (!scale.range.every((v) => isFinite(v))) return false;
```

Comment:
Minor - use `Number.isFinite` as `window.isFinite` will try to convert strings and such to numbers first